src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Public.Data/PreviewTokenService.cs

```diff
@@ -152,8 +152,8 @@ private async Task<PreviewTokenViewModel> MapPreviewToken(PreviewToken previewTo
             Status = previewToken.Status,
             CreatedByEmail = createdByEmail,
             Created = previewToken.Created,
-            Activates = previewToken.Activates ?? previewToken.Created,
-            Expiry = previewToken.Expiry,
+            Activates = previewToken.Activates,
+            Expires = previewToken.Expires,
```

Comment:
Highlighting a change to `Activates` which has crept into this PR. Not sure if this is intentional or not.